k4e-agent-sti-upgrade.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
+
+set -e
+
+# The script performs the following steps on rhel-8.5:
+# * Create new rpms for yggdrasil, k4e-device-worker and Prometheus node_exporter
+# * Publish rpms as yum repo to be consumed by image-builder
+# * Publish rhel4edge image commit for edge-device
+# * ./k4e-agent-sti-upgrade.sh -a <server_adress> -i <image_name> -o helios06.lab.eng.tlv2.redhat.com:8888 -v
```

Comment:
let's replace specific host name with placeholder / alias (but not a concrete name)